tests/Jobs/ElasticSearchIndexDeleteTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\User;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+use App\Http\Curl\HttpRequest;
+use App\WikiManager;
+use App\WikiSetting;
+use App\Wiki;
+use App\Jobs\ElasticSearchIndexDelete;
+
+class ElasticSearchIndexDeleteTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    private $wiki;
+    private $user;
+
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->user = User::factory()->create(['verified' => true]);
```

Comment:
If we start using the dbname, these tests will also have to create a wikiDB entry.

Reply:
done